### PR TITLE
Audit members of our Trello boards

### PR DESCRIPTION
**Pull request by bjgill**
Trello: https://trello.com/c/bHA7PWRG/2156-audit-all-people-with-access-to-dmp-trello-boards

Create a script that uses the Trello API to get information on the members of all our boards.

We need this because we currently have people who have access to our boards who should not have access.

### CODE REVIEW

**Review by lfdebrux — APPROVED** [review]

**Comment by lfdebrux** [comment]
I wonder if we should have a default or global list of organisation IDs somewhere, maybe in digitalmarketplace-credentials? Just so the output depends less on who is running the script.

**Comment by bjgill** [comment]
@lfdebrux - good idea. The organisation IDs are not secret, so I think we're fine to put them in the script - you'd need an API key and token for someone with access to do anything with the IDs.

File: scripts/get_trello_members.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        headers=HEADERS,
+        params={"member": "true", **QUERY},
+    )
+    response.raise_for_status()
+
+    for person in json.loads(response.text):
+        members.setdefault(person["member"]["username"], []).append(board_name)
+
+print(members.keys())
+print(members)
```
Comment (lfdebrux) [inline scripts/get_trello_members.py:63]: This probably should be behind an `if ... "__main__"` as I just tried to import the script and was surprised when it started running 😅

File: scripts/get_trello_members.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+def get_organisations():
+    """
+    All the DMp Trello organisations.
+    """
+    return {
+        "DMP 2": "5e70a808f51fba20f624d9e2",  # pragma: allowlist secret
+        "DMP 3": "5eb0095a4b182882dbb46652",  # pragma: allowlist secret
+        "DMP 4": "5f11bf47fb7c2f529c42193a",  # pragma: allowlist secret
+        "Digital Marketplace - up to May 2019": "56ab94d6239d393aafcc0dd7",  # pragma: allowlist secret
+        "Digital Marketplace Missions": "5cd16b6993f4f0616b48c101",  # pragma: allowlist secret
```
Comment (lfdebrux) [inline scripts/get_trello_members.py:26]: Some additional orgs:

```suggestion
        "Digital Marketplace Missions": "5cd16b6993f4f0616b48c101",  # pragma: allowlist secret
        "Digital Marketplace UR": "5cc18c22ae0b160c5229e07e",  # pragma: allowlist secret
        "DMP EMPATHY LAB": "5e99bd017496273606e74187",  # pragma: allowlist secret
```

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**